tools/assets/build.sh

```diff
@@ -49,8 +49,8 @@ build_asset() {
       build "$PROJECT_PATH/themes/classic/_dev"
     ;;
     all)
-      build_asset admin-default
       build_asset admin-new-theme
+      build_asset admin-default
```

Comment:
Not sure it changes much things but why not
Unless this is now mandatory because the create product bundle must be built before the default theme? Since bundles are generated separately and included in the layout it shouldn't matter right?

Reply:
yeah, because default theme uses new theme JS it failed if the new theme was not setup before it. So thats why I changed it.